packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -0,0 +1,246 @@
+* [Custom URL domain](/docs/guides/custom-url-domain/main/)
+
+**Sample code**
+
+[Customization examples](#customization-examples)
+
+---
+
+## About the Sign-In Widget (third generation)
+
+The Sign-In Widget enables the following:
+
+* Registration
+* Enrollment
+* Verification
+* Recovery experiences for your users
+
+It's where everyone in your org starts their Okta session.
+
+The third generation of the Sign-In Widget offers the same user experience as the second generation. It adds accessibility improvements in the following:
+
+* Color contrast
+* Focus management
+* Screen reader behavior
+
+See [Sign-In Widget, third generation](https://help.okta.com/okta_help.htm?type=oie&id=ext-compare-siw) to verify if the third generation is right for your org.
+
+## Upgrade to the third generation
+
+> **Note:** If you have a new Okta Identity Engine org, the third-generation Sign-In Widget isn't enabled by default.
+
+To upgrade to the third generation:
+
+1. In the **Admin Console**, go to **Customizations** > **Brands**.
+1. Select a brand.
+1. Go to the **Pages** tab, and then click **Edit** or **Configure** for the **Sign-in page**.
+1. Click the **Settings** tab.
+1. In the **Sign-In Widget version** section, click **Edit**.
+1. To use the third generation, click the **Use the third generation** toggle so that it's active. Leave the toggle inactive to remain on the second generation.
+1. Click **Save to draft**.
+
+See [Style the sign-in page](/docs/guides/custom-widget/main/) for details about customizing the second generation.
+
+## Use the code editor
+
+Use the code editor to add design tokens to your code. Design tokens call a JavaScript API.
+
+> **Note:** You can only enable the code editor if you configure a [custom domain](/docs/guides/custom-url-domain/).
+
+See [Customization examples](#customization-examples) for snippets that you can update and use.
+
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
+2. In the **Pages** tab, click **Configure** in the **Sign-in page** panel.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+4. Make changes directly in the editor. See [Use design tokens](#use-design-tokens).
+   * Click **Save to draft**, then **Preview**, **Revert**, or **Publish**.
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
+
+   > **Note:** To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. Turning off the code editor restores the default JavaScript code.
+
+## Use design tokens
+
+Design tokens make the Sign-In Widget's visual style consistent and easier to update. Tokens replace static values to customize the following:
+
+* color
+* font
+* line height
+* size
+* tracking
+* weight
```

Comment:
Not sure what "tracking" means. FYC:

* colors
* borders
* font family
* font size
* font weight
* line height
* spacing